From 16e992a59fd0f1995efba4b7f91b0d549b5ea63a Mon Sep 17 00:00:00 2001
From: Guilherme Costa <guilherme.costa@windriver.com>
Date: Wed, 6 Mar 2024 17:28:50 -0300
Subject: [PATCH] csm-observability: change CSIs secret name

This patch consists of changing the secretName used from csi-powerstore,
csi-powerflex, csi-powermax and csi-powerscale.
It's needed because the secret of the CSIs are created based on
the default release name and it were used hard-coded here.

Signed-off-by: Guilherme Costa <guilherme.costa@windriver.com>
---
 .../templates/karavi-metrics-powerflex.yaml                   | 4 ++--
 .../templates/karavi-metrics-powermax.yaml                    | 2 +-
 .../templates/karavi-metrics-powerscale.yaml                  | 4 ++--
 .../templates/karavi-metrics-powerstore.yaml                  | 2 +-
 4 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/charts/karavi-observability/templates/karavi-metrics-powerflex.yaml b/charts/karavi-observability/templates/karavi-metrics-powerflex.yaml
index 177b3d1..84acd43 100644
--- a/charts/karavi-observability/templates/karavi-metrics-powerflex.yaml
+++ b/charts/karavi-observability/templates/karavi-metrics-powerflex.yaml
@@ -103,7 +103,7 @@ spec:
       volumes:
       - name: vxflexos-config
         secret:
-          secretName: vxflexos-config
+          secretName: csi-powerflex-config
       - name: tls-secret
         secret:
           secretName: otel-collector-tls
@@ -117,7 +117,7 @@ spec:
      {{- if eq .Values.karaviMetricsPowerflex.authorization.enabled true }}
       - name: karavi-authorization-config
         secret:
-          secretName: karavi-authorization-config
+          secretName: csi-powerflex-authorization-config
       - name: proxy-server-root-certificate
         secret:
           secretName: proxy-server-root-certificate
diff --git a/charts/karavi-observability/templates/karavi-metrics-powermax.yaml b/charts/karavi-observability/templates/karavi-metrics-powermax.yaml
index 31891ab..7ac8729 100644
--- a/charts/karavi-observability/templates/karavi-metrics-powermax.yaml
+++ b/charts/karavi-observability/templates/karavi-metrics-powermax.yaml
@@ -123,7 +123,7 @@ spec:
      {{- if eq .Values.karaviMetricsPowermax.authorization.enabled true }}
       - name: karavi-authorization-config
         secret:
-          secretName: powermax-karavi-authorization-config
+          secretName: csi-powermax-authorization-config
       - name: proxy-server-root-certificate
         secret:
           secretName: powermax-proxy-server-root-certificate
diff --git a/charts/karavi-observability/templates/karavi-metrics-powerscale.yaml b/charts/karavi-observability/templates/karavi-metrics-powerscale.yaml
index 510e946..77d44fe 100644
--- a/charts/karavi-observability/templates/karavi-metrics-powerscale.yaml
+++ b/charts/karavi-observability/templates/karavi-metrics-powerscale.yaml
@@ -103,7 +103,7 @@ spec:
       volumes:
       - name: isilon-creds
         secret:
-          secretName: isilon-creds
+          secretName: csi-powerscale-creds
       - name: tls-secret
         secret:
           secretName: otel-collector-tls
@@ -117,7 +117,7 @@ spec:
      {{- if eq .Values.karaviMetricsPowerscale.authorization.enabled true }}
       - name: karavi-authorization-config
         secret:
-          secretName: isilon-karavi-authorization-config
+          secretName: csi-powerscale-authorization-config
       - name: proxy-server-root-certificate
         secret:
           secretName: isilon-proxy-server-root-certificate
diff --git a/charts/karavi-observability/templates/karavi-metrics-powerstore.yaml b/charts/karavi-observability/templates/karavi-metrics-powerstore.yaml
index a54d1d5..8fcd046 100644
--- a/charts/karavi-observability/templates/karavi-metrics-powerstore.yaml
+++ b/charts/karavi-observability/templates/karavi-metrics-powerstore.yaml
@@ -76,7 +76,7 @@ spec:
               path: localhost.key
       - name: powerstore-config
         secret:
-          secretName: powerstore-config
+          secretName: csi-powerstore-config
       - name: tls-secret
         secret:
           secretName: otel-collector-tls
--
2.25.1
